src/content/docs/iast/troubleshooting.mdx

```diff
@@ -0,0 +1,289 @@
+---
+title: IAST Troubleshooting
+tags:
+    - IAST
+    - New Relic IAST
+    - Troubleshooting
+metaDescription: Find out how to solve your New Relic IAST issues.
+freshnessValidatedDate: never
+---
+
+This is the first step in diagnosing your New Relic IAST problem yourself, use this guide to resolve issues quickly. 
+
+If you continue to have trouble, you can always reach out to [New Relic Support](https://support.newrelic.com) if you don't find your problem here.  
```

Comment:
I'd say "if you don't find your problem listed here" or something along those lines, for clarity